tonic-web/src/service.rs

```diff
@@ -255,6 +256,27 @@ mod tests {
         const NAME: &'static str = "test";
     }
 
+    fn enable<S>(service: S) -> Layered<tower_http::cors::Cors<GrpcWebService<S>>, S>
+    where
+        S: Service<http::Request<BoxBody>, Response = http::Response<BoxBody>>,
+    {
+        tower_layer::Stack::new(
+            crate::GrpcWebLayer::new(),
+            tower_http::cors::CorsLayer::new(),
+        )
+        .named_layer(service)
+    }
+
+    fn get_name_of_named_service<S: NamedService>(_: &S) -> &'static str {
+        S::NAME
+    }
+
+    #[test]
+    fn named_service_is_propagated_by_enabled_helper_function() {
```

Comment:
What are we trying to test here? I don't see the value of this test (especially considering that it needs the trivial utility function).